subsys/spm/secure_services.c

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+#include <cortex_m/tz.h>
+
+#if defined(CONFIG_ARM_FIRMWARE_HAS_SECURE_ENTRY_FUNCS)
+
+extern void sys_arch_reboot(int type);
+
+/** @brief Implementation of Secure System Reset
+ *
+ * Currently, this simply calls the Zephyr System Reset function,
+ * issuing a system reboot immediately.
+ *
+ * Note: the function will be located in a Non-Secure
+ * Callable region of the Secure Firmware Image.
+ */
+static void spm_system_reboot(void)
+{
+	sys_arch_reboot(0);
```

Comment:
sys_reboot seem more appropriate (and it also calls sys_arch_reboot under the hood)

Reply:
We use spm_ as a prefix for the functions in the SPM subsystem

Reply:
@frkv done.
@ioannisg I think he meant to call `sys_reboot` instead of `sys_arch_reboot` in line 22.